docs/trust-list-specification.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 4
+---
+
+# Trust List Specification
+
+## Introduction
+In order for data to be exchanged between different COVID-19 credential standards, there needs to be a means of unifying the trust lists of each respective specification. A Global Trust Network has the opportunity to bring existing and new local and regional trust networks together in a way that respects their designs and sovereignty while contributing to real interoperability among all of them.
+
+## Objective and Goals
+Finding alignment between existing trust list formats is the key to real interoperability. This can be done by defining a common trust list format to assemble and share public key infrastructure (PKI) for all COVID-19 credential specifications used by existing trust networks.
+
+## Requirements
+A common trust list specification should define the lowest common denominator format that can interoperate between all specifications in the trust list. This includes considering the minimum security requirements that satisfy each of the specifications. Creation of the trust list specification will require consideration for how to define a structure that can support the minimal required features from the included specifications. To enumerate the requirements, a trust list specification format must include the following tenets:
+1. SHALL be convertible from each existing trust network's formats
+2. SHALL describe a key-to-trust-anchor path for all specifications
+3. SHALL be cacheable
+4. SHALL be mergeable (trust list operators can integrate each other's entries)
+5. SHALL be integrable with GCCN's metadata format
```

Comment:
```suggestion
5. SHALL be integrable with the Digital TRUST Infrastructure for Discovery and Validation's (Regi-TRUST's) metadata format
```

Reply:
This GCCN-specific tenet was removed in https://github.com/dvci/cc-pki/pull/9/commits/8b2e2c8c6967ee1be7613727e5717f49685c1494#diff-c3cc675368d2ab3fcd9e3750a5e781ce5cb26bed90ccc252e4f1ed0e9c9c9e7eL19